airflow/docker.go

```diff
@@ -656,7 +657,11 @@ var checkWebserverHealth = func(settingsFile string, project *types.Project, com
 					}
 				}
 				return errComposeProjectRunning
+			} else if string(marshal) == `{"action":"health_status: unhealthy"}` {
```

Comment:
currently on M1 mac, the webserver will always go to an unhealthy state and then become healthy, this is because it takes about 4-5 minutes of start-up time, and our current health check configuration isn't enough to wait for such long. So with these changes on M1 mac, it will always go to this else clause even if the webserver most likely would come up after a few minutes.

So we might want to increase the health check retries over here in this PR: https://github.com/astronomer/astro-cli/blob/main/airflow/include/composeyml.go#L99-L104

Reply:
what should we increase the health check retries too?

Reply:
I would say 4-5 minutes is a long time to wait especially if you are watching the screen. It currently waits 30 seconds, so how does bumping the retries to 45 sound? That gives us 1 and a half minutes.

Reply:
But the problem with having 1:30 minutes is that on M1 mac it is never going to be enough at this point in time, thus always sending back the error response that the project is not running, which means we would be conveying incorrect information back to the user as the project is running and trying to start up and cause confusion.

Hence in trying to solve one issue, we might end up causing another issue from the user's POV

Reply:
Or if we can be somewhat smarter, then we can have separate health check logic for separate machine types, i.e. if a user is on a darwin arm64 machine then we have a health check of 5 minutes, else we have a health check of 60 seconds. wdyt? cc @sunkickr @jemishp